src/components/sharedStyle.css

```diff
@@ -84,3 +84,16 @@
 .text-align-left {
   text-align: left;
 }
+
+.selectBox-has-error {
+    border-color: #cc0000;
+}
```

Comment:
Maybe we could use already existing PF styles https://www.patternfly.org/v3/styles/color-palette/
Or is it better to add the appropriate border colors like this? I'm not sure. WDYT?

Reply:
we chatted about that on slack, so I'll mark it as resolved

Reply:
> we chatted about that on slack

A comment in the code with summary would be nice:)

Anyway ...this rule gets overridden by on-hover rule. Built-in styles should support all possible variations i.e. error and on-hover, error and selected, error and disabled etc